test/functional/Jsr292/src/com/ibm/j9/jsr292/indyn/BootstrapMethods.java

```diff
@@ -45,6 +45,17 @@
 import differentpackage.CrossPackageHelper;
 
 public class BootstrapMethods {
+
+	// Bootstrap method throws null the first time, else a valid CallSite
+	private static int bootstrap_test_CallSiteNullErrorRethrown_counter = 0;
+	public static CallSite bootstrap_test_CallSiteNullErrorRethrown( Lookup lookup, String name, MethodType type ) throws Throwable {
+		if (0 == bootstrap_test_CallSiteNullErrorRethrown_counter) {
+			bootstrap_test_CallSiteNullErrorRethrown_counter++;
+			return null;
+		}
+		MethodHandle handle = MethodHandles.lookup().findStatic( Helper.class, "voidMethod", methodType( void.class ) );
```

Comment:
There's a Lookup passed into the BSM so we don't need to create another.
```suggestion
		MethodHandle handle = lookup.findStatic( Helper.class, "voidMethod", methodType( void.class ) );
```